test/builtin-storage-class.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck --check-prefix=CHECK-SPIRV %s
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+
+; CHECK-SPIRV: Name [[BuiltinId:[0-9]+]] "__spirv_BuiltInWorkgroupId"
+; CHECK-SPIRV-DAG: Decorate [[BuiltinId]] BuiltIn
+; CHECK-SPIRV-DAG: TypePointer [[PointerTypeId:[0-9]+]] 1
```

Comment:
the number '1' corresponds to StorageClassInput here. Thanks

Reply:
I think this comment should be put in the source.

Reply:
Added this info in header comment. Thanks